types/yargs/index.d.ts

```diff
@@ -286,7 +286,7 @@ declare namespace yargs {
          * Inside `cmd`, the string `$0` will get interpolated to the current script name or node command for the present script similar to how `$0` works in bash or perl.
          * Examples will be printed out as part of the help message.
          */
-        example(command: string, description: string): Argv<T>;
+        example(command: string | ReadonlyArray<[string, string?]>, description: string): Argv<T>;
```

Comment:
Looking at the documentation http://yargs.js.org/docs/#api-reference-examplecmd1-desc1-cmd2-desc2 I don't think the second string param is valid with the array form.

ie:
```
example(command: string, description: string): Argv<T>;
example(commandsAndDescriptions: ReadonlyArray<[string, string?]>): Argv<T>;
```

Reply:
Ah yeah -- looks like yargs defines this as `example (cmd: string | [string, string?][], description?: string): YargsInstance` -- will update

Reply:
That's the internal interface, but the docs imply that if you pass in a string array, the second param will be ignored.  Intuitively that makes sense because what would that be describing?

Reply:
Sure, I'll update the definition to be two different function calls since the second description param is ignored as indicated in the docs